src/debug.c

```diff
@@ -658,7 +658,9 @@ void debugCommand(client *c) {
         sds s = sdsempty();
         s = sdscatprintf(s, "Value at:%p refcount:%d encoding:%s", (void *)val, val->refcount, strenc);
         if (!fast) s = sdscatprintf(s, " serializedlength:%zu", rdbSavedObjectLen(val, c->argv[2], c->db->id));
+        /* Either lru or lfu field could work correctly which depends on server.maxmemory_policy. */
         s = sdscatprintf(s, " lru:%d lru_seconds_idle:%llu", val->lru, estimateObjectIdleTime(val) / 1000);
+        s = sdscatprintf(s, " lfu_freq:%d lfu_access_time_minutes:%u", val->lru & 255, val->lru >> 8);
```

Comment:
somehow i think the lfu_freq should use LFUDecrAndReturn? what do you think?
```suggestion
        s = sdscatprintf(s, " lfu_freq:%d lfu_access_time_minutes:%u", LFUDecrAndReturn(val), val->lru >> 8);
```

Reply:
I think `debug` command here should be like a "NOTOUCH" command, which means when you call this command, nothing should be changed.

Like lru, `estimateObjectIdleTime()` will not change the lru time belonging to the object, so I would choose not to update lfu here.

Reply:
LFUDecrAndReturn did not changed anything, right? updateLFU do the update

Reply:
YES you are right! I missed that.
I check another command `object freq` and that also returns freq with `LFUDecrAndReturn`, so let's use it.